pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/sierramets/SierraMetsWorkPairMerger.scala

```diff
@@ -0,0 +1,55 @@
+package uk.ac.wellcome.platform.merger.rules.sierramets
+
+import uk.ac.wellcome.models.work.internal.{
+  DigitalLocation,
+  Identifiable,
+  IdentifiableRedirect,
+  Item,
+  Location,
+  TransformedBaseWork,
+  Unidentifiable,
+  UnidentifiedRedirectedWork,
+  UnidentifiedWork
+}
+import uk.ac.wellcome.platform.merger.model.MergedWork
+import uk.ac.wellcome.platform.merger.rules.{MergerRule, WorkPairMerger}
+
+trait SierraMetsWorkPairMerger extends WorkPairMerger {
```

Comment:
Might help having a scaladoc comment on this trait, with a brief summary of the merging logic within.

Reply:
I've just added it